templates/csharp/sso-tab/Components/Welcome.razor

```diff
@@ -11,8 +11,8 @@
 @if(isLoading)
 {
     <div style="display: flex; justify-content: center; align-items: center; min-height: 100vh;">
-		<FluentProgressRing/>
-	</div>
+        <FluentProgressRing/>
```

Comment:
Is this spaces or tab?

Reply:
Because I formatted the file all the way through, there will be formatting inconsistencies.

Reply:
As best practice, it is better to use space instead of tab in source code for better compatibility.